win_build.ps1

```diff
@@ -119,6 +119,7 @@ if (-Not $skipExporterCompile)
 
     Pop-Location
     Copy-Item "$env:GOPATH\src\$exporterRepo\output\$arch\$exporterBinaryName" -Destination ".\target\bin\windows_$arch\" -Force 
+    Copy-Item "$env:GOPATH\src\$exporterRepo\LICENSE" -Destination ".\target\bin\windows_$arch\" -Force 
```

Comment:
Would this make clear that this is a third-party license rather than of our own? Perhaps we could append `-$exporterRepo` to the end of the file to clarify that.

Reply:
I would follow the very same naming convention chose in the "exporter package repo" (however, I do not remember which it was :) )

Reply:
good call, yes we used `$exporterRepo-LICENSE` I'll do that